meshio/mesh.py

```diff
@@ -4,12 +4,21 @@
 
 
 class Mesh(object):
-    def __init__(self, points, cells, point_data=None, cell_data=None, field_data=None):
+    def __init__(
+        self,
+        points,
+        cells,
+        point_data=None,
+        cell_data=None,
+        field_data=None,
+        nsets=None,
```

Comment:
Let's be more verbose here and call it `node_sets`.

Reply:
OK. Modifing it!